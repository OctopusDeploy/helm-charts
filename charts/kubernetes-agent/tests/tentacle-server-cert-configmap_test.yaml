suite: tentacle configuration
templates:
  - templates/tentacle-server-cert-configmap.yaml
tests:
  - it: "is not created if no encodedCertificate is provided"
    asserts:
    - hasDocuments:
        count: 0        
  - it: "matches snapshot"
    set:
      agent:
        serverCA:
          encodedCertificate: "dGVzdC10ZXh0"
    asserts:
      - matchSnapshot: {}

