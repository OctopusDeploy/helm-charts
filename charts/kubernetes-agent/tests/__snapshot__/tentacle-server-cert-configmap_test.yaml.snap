matches snapshot:
  1: |
    apiVersion: v1
    data:
      octopus-server-certificate.pem: test-text
    kind: ConfigMap
    metadata:
      name: octopus-server-certificate
      namespace: NAMESPACE
